filebeat/_meta/config/filebeat.autodiscover.reference.yml.tmpl

```diff
@@ -14,3 +14,26 @@
 #            - type: container
 #              paths:
 #                - /var/log/containers/*.log
+
+#Example: for kubernetes container logs autodiscovery
+#  filebeat.autodiscover:
+#    providers:
+#      - type: kubernetes
+#        node: ${NODE_NAME}
+#        hints.enabled: true
+#        use_kubeadm: true
```

Comment:
```suggestion
#        # By default requests to kubeadm config map are made in order to enrich cluster name by requesting /api/v1/namespaces/kube-system/configmaps/kubeadm-config API endpoint.
#        use_kubeadm: true
```

would it be better to move this description on top of the config parameter, instead of adding it to the end?
the same comments for other similar files